opaque_keys/edx/locator.py

```diff
@@ -48,7 +47,7 @@ class Locator(OpaqueKey):
     ALLOWED_ID_CHARS = r'[\w\-~.:]'
     DEPRECATED_ALLOWED_ID_CHARS = r'[\w\-~.:%]'
 
-    @abstractproperty
+    @property
```

Comment:
why changing this from `abstractproperty` to `property`

Reply:
abstractproperty is depracated since Python 3.3. This was a failing in quality.

Reply:
Add documentation link.

Reply:
https://docs.python.org/3/library/abc.html#abc.abstractproperty